mods/ra/maps/training-camp/map.yaml

```diff
@@ -1058,9 +1058,10 @@ VoxelSequences:
 
 Weapons:
 	CrateNuke:
-		Warhead@areanuke2: PerCellDamage
+		Warhead@areanuke2: SpreadDamage
 			Damage: 250
-			Size: 4
+			Spread: 1c0
+			Falloff: 100, 100, 100, 100, 100, 0
```

Comment:
Maybe port and rebalance the proper nuke warhead, instead of trying to recreate the old behaviour?